roqoqo/Cargo.toml

```diff
@@ -66,3 +66,4 @@ json_schema=["schemars", "serialize", "qoqo_calculator/json_schema", "jsonschema
 circuitdag = ["petgraph"]
 unstable_chain_with_environment = []
 unstable_spin_boson_operations = []
+unstable_analog_operations = []
```

Comment:
Do you need to use the roqoqo-derive unstable trait here, as well as in its import above?

Reply:
I am not sure what you mean by this